### ISSUE TITLE
[Feature] Dynasty 8 Real Estate Requests

### ISSUE THREAD

**Issue by theCitizen420** [FrenzyRoleplay/server-issues#6]
### Is your feature request related to a problem? Please describe.
At this time, the Dynasty 8 Real Estate Office is under-utilized... or nearly never utilized. In addition to the update requests at Dynasty 8, there are additional updates requested on behalf of the citizens.

### Describe the solution you'd like
**Updates to the Dynasty 8 Real Estate Office:**
1. Company cars - 2x Baller LEs and 2x Boxvilles (for moving services we can offer for our clients).
2. Speaking of the logos, it would be AMAZING to have the White Space above my Secretary's Desk to say Dynasty 8 or Dynasty 8 Real Estate. It's basically a perfect spot for it. 
3. Add the ability to check public records as a realtor at the Real Estate Office (at the main desk computer) to enable us to enforce 1 property per person PLUS (?) being able to check their credit history (maybe a line on their public record entry saying "Delinquent Credit". 
4. Install a realtor-only garage in the bottom of the Arcadius Center where a car lift already exists. When we park, we can be "brought" into the office and have a place inside the office where we can request our car -- maybe the other set of elevators in the hallway before we enter the office as they're unused anyways. 
5. Add an outfit changing area as we already have an office that supports it with the dressing type room on the side. This ensures my agents aren't caught off guard and dressed like hooligans. Plus it's basically a locker room already in there. 
6. In addition to the Safe where money is deposited, could a small stash be added for us to store "Welcome Home Packages" which would more or less be items we give to the new homeowners (like a gift). We'd also like to store our coupons and other company items. This would also be in the locker room. The add-on reasoning here is that we have a fully setup bar so it would be nice to have alcohol and things of the sort available to actually drink, not just /e whiskey. It's okay to /e whiskey, but using a whiskey is way more satisfying. 
7. Change the #deposit-log to the #dynasty-log and add the following to the log itself (if possible please as they are impossible to track at this time):
- Transfers including agent who did it, person it goes to, and ID of property
- Updatehouse command including agent who did it and ID of property
8. Add the ability to store boats in garages - for example the Canals has houses with boat docks. The owner can opt for a boat garage or a car garage (only one or the other).

### Describe alternatives you've considered
Alternatives are essentially baked into the above, or did not have an alternative option to offer. 

### Additional context
Happy to attach screenshots with any supporting information you may want.

**Comment by theCitizen420** [comment]
In addition to the requests above:

10. Update the Newspaper with current Realtors:
-  Remove Bella Bordeaux and Luther Kingsley
- Add Zach Lance 661-871-7400
- Add Iggy Karpalopagus 310-374-5627

11. Change the #deposit-log to the #dynasty-log and add the following to the log itself (if possible please as they are impossible to track at this time):
- Transfers including agent who did it, person it goes to, and ID of property
- Updatehouse command including agent who did it and ID of property

**Comment by theCitizen420** [comment]
Hey BadCodes,
I scoped out logical apartment complexes or areas where simple apartments are useful (like the barrio). I've identified 10 locations where 5 of them are in the South Side. The testing was completed with Interior 5 (trailer) as the example and went as follows:
- Put two T5 to the left and right of each other. You will see others in your own home if you put the house door "too close". This means we would spread them out by the size of the chosen interior. This will work as long as the managing realtor does their job right. I can restrict this to myself in terms of creating the homes to ensure less hiccups along the way. This was completed in all locations identified (South Side, Paleto, Del Perro Beach, etc.)
- Put two T5 on top of each other and you will see foot clipping in the ceiling from the lower home. This will not work and was expected. This was attempted once, then I stopped bothering. No reason to push the envelope as I never intended this to work. 

We could pilot out the South Side for now if you were open to this idea? If I could see what housing interiors are available then I could make a recommendation myself. I've been interested in learning how to do this type of work (and found some guides, so just doing research in that regard). 

![SouthSideApartments](https://user-images.githubusercontent.com/76059658/106190812-29315200-6178-11eb-927d-445e830fb7c4.JPG)
![PaletoApartments](https://user-images.githubusercontent.com/76059658/106190814-29315200-6178-11eb-8035-ec543bbd43cd.JPG)
![RockfordVespucciApartments](https://user-images.githubusercontent.com/76059658/106190815-29315200-6178-11eb-8312-343bbdc90ec5.png)
